modules/openxr/openxr_interface.cpp

```diff
@@ -1408,6 +1434,24 @@ Vector3 OpenXRInterface::get_hand_joint_angular_velocity(Hand p_hand, HandJoints
 	return Vector3();
 }
 
+RID OpenXRInterface::get_vrs_texture() {
+	if (!openxr_api) {
+		return RID();
+	}
+
+	PackedVector2Array eye_foci;
+
+	Size2 target_size = get_render_target_size();
+	real_t aspect_ratio = target_size.x / target_size.y; // is this y/x ?
```

Comment:
Since things seem to be working, I assume this comment is unneeded? Or, if there still is an open question on which ratio is correct, we should probably figure that out :-)

```suggestion
	real_t aspect_ratio = target_size.x / target_size.y;
```